proto/pulumi/errors.proto

```diff
@@ -23,3 +23,12 @@ message ErrorCause {
     string stackTrace = 2;
 }
 
+// An error that can be returned from a component provider and includes details of the
+// error, which can be multiple properties.
+message InvalidInputProperties {
+    message InvalidInputProperty {
+        string property_name = 1;
```

Comment:
Should this be property_name or property_path? Feels like the latter is probably what providers will often want.

Reply:
Yeah `property_path` makes more sense, fixed.